lib/src/core/message/reaction.dart

```diff
@@ -15,6 +21,10 @@ abstract class IReaction {
 
 /// Reaction object. [emoji] field can be partial [GuildEmoji].
 class Reaction implements IReaction {
+  /// Reference to [NyxxWebsocket]
+  @override
+  late final INyxx client;
```

Comment:
The `late` here isn't necessary.

Reply:
It is, otherwise dart will complain that client has not been initialized.
That's due to the [`Reaction.event()`](https://github.com/nyxx-discord/nyxx/blob/68ebcde5c7fdf37a5dacb71777c02de988ad9afc/lib/src/core/message/reaction.dart#L45) method below.

Reply:
Ah, I missed that. Thanks for pointing that out!